### ISSUE TITLE
ECS Editor - missing modules

### ISSUE THREAD

**Issue by ThaDoctor72** [Pinacolada64/ImageBBS#31]
On the R2 disk (that I'm using) there were a few plus files missing from the distro.  While this is fine, if we're supposed to obtain them by other means, the existing ecs data on the distro should be modified to have them removed from the list. 

+.TU
+.CD
+.word
+.NM/chess
+.OR
+.NW
+.TAG

**Comment by x-tec2017** [comment]
Yes, there are commands in the ECS list that had no modules included in the files that I originally received. If an ECS command has a red * next to the command name, it means it is inactive. I could have removed the commands without modules included but I left them in there as an indicator of what the original programmers had plans for and items that might possibly be added in the future.

**Comment by Pinacolada64** [comment]
> No idea of what +.CD, and +.TAG were planned for.

I'm betting `+.CD` was the "change device address" program, renumbered from 1.2.

Maybe `+.TAG` was a shortcut for `+.test select`?

**Comment by Pinacolada64** [comment]
I found `+.word` from a 1.2 pfile disk, any interest in updating it to 2.0? It's a user dictionary type thing. I never ran it, but it might be interesting.
